vanilla_installer/gtk/widget-partition.ui

```diff
@@ -1,49 +1,236 @@
+            <child type="suffix">
+              <object class="GtkButton" id="launch_gparted">
+                <property name="icon-name">adw-external-link-symbolic</property>
+                <property name="valign">center</property>
+                <property name="halign">center</property>
+                <style>
+                  <class name="circular"/>
+                </style>
+              </object>
+            </child>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="boot_part">
+        <property name="title" translatable="true">Boot Partition</property>
+        <property name="description" translatable="true">The partition to mount under “/boot”. This directory stores files for booting in Vanilla OS.</property>
+        <child>
+          <object class="AdwExpanderRow" id="boot_part_expand">
+            <property name="title" translatable="true">No partition selected</property>
+            <property name="subtitle" translatable="true">Please select a partition from the options below</property>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="boot_small_error">
+        <style>
+          <class name="error"/>
+        </style>
+        <property name="visible">false</property>
+        <property name="title" translatable="true">Boot partition is too small.</property>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="efi_part">
+        <property name="title" translatable="true">EFI Partition</property>
+        <property name="description" translatable="true">The partition to mount under “/boot/efi”. This directory stores files for booting in the bootloader.</property>
+        <child>
+          <object class="AdwExpanderRow" id="efi_part_expand">
+            <property name="title" translatable="true">No partition selected</property>
+            <property name="subtitle" translatable="true">Please select a partition from the options below</property>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="efi_small_error">
+        <style>
+          <class name="error"/>
+        </style>
+        <property name="visible">false</property>
+        <property name="title" translatable="true">EFI partition is too small.</property>
+      </object>
+    </child>
+    <child>
+      <object class="AdwPreferencesGroup" id="bios_part">
+        <property name="title" translatable="true">BIOS Boot Partition</property>
+        <property name="description" translatable="true">A partition required for all BIOS systems running in GPT-formatted disks. This partition must be located in the first 2 TB of the disk and must be exactly 1 Mebibyte (1 MiB).</property>
```

Comment:
```suggestion
        <property name="description" translatable="true">A partition is required for all BIOS systems running in GPT-formatted disks. This partition must be located in the first 2 TB of the disk and must be exactly 1 Mebibyte (1 MiB).</property>
```

Reply:
This is a description for what is a BIOS partition, so I'm not sure about adding an "is" there. Maybe we could change to something like "The partition required by GPT-formatted disks to run on BIOS systems..."

Reply:
Yeah, that is better